<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingForm</class>
 <widget class="QWidget" name="SettingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="txGridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="5" column="0">
    <spacer name="txLeftHorizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="2">
    <layout class="QHBoxLayout" name="txBottomHorizontalLayout">
     <item>
      <widget class="QPushButton" name="txbtn_save">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="txHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="txbtn_reset">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="10">
    <spacer name="txRightHorizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="txTopVerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="2">
    <spacer name="txBottomVerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="txtb_address"/>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="txtb_account"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="txtb_password"/>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="txtb_dirpath"/>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="txtb_timeout"/>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="txtb_interval"/>
   </item>
   <item row="10" column="2">
    <widget class="QLineEdit" name="txtb_threadnum"/>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="txWeblibTimeoutSecLabel">
     <property name="text">
      <string>秒</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="txScanIntervalSecLabel">
     <property name="text">
      <string>秒</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <spacer name="txMiddleVerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="txWeblibAddrLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Weblib地址：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="txWeblibAccountLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>账号：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="txWeblibPassLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="txLocalDirLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>上传目录：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="txLocalDirLavelLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>目录深度</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="txWeblibDirLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Weblib上传位置：</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="txWeblibTimeoutLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>超时设置：</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="txUploadSpeedLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>上传速度限制：</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="txUploadThreadCountLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>上传线程数：</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="txScanIntervalLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>文件扫描间隔：</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QSpinBox" name="txtb_dirlevel"/>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="txbtn_dirselect">
     <property name="text">
      <string>选择</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="txWeblibAddr2Label">
     <property name="text">
      <string>http://开头</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="txtb_groupname">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkb_userdir">
       <property name="text">
        <string>创建用户目录</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="txbtn_group">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>选择</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="txtb_speed"/>
     </item>
     <item>
      <widget class="QLabel" name="txUploadSpeedKbLabel">
       <property name="text">
        <string>kB/s</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>等于0则不限速</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="weblibuploader.qrc"/>
 </resources>
 <connections/>
</ui>
